src/borg/testsuite/archiver/diff_cmd.py

```diff
@@ -170,14 +172,17 @@ def get_changes(filename, data):
                 assert {"type": "removed link"} in get_changes("input/link_removed", joutput)
 
                 # Symlink replacing or being replaced
-                assert any(
-                    chg["type"] == "mode" and chg["new_mode"].startswith("l")
-                    for chg in get_changes("input/dir_replaced_with_link", joutput)
-                )
-                assert any(
-                    chg["type"] == "mode" and chg["old_mode"].startswith("l")
-                    for chg in get_changes("input/link_replaced_by_file", joutput)
-                )
+
+                # TODO: Add mode test flag to assert function
+                if not content_only:
+                    assert any(
+                        chg["type"] == "mode" and chg["new_mode"].startswith("l")
+                        for chg in get_changes("input/dir_replaced_with_link", joutput)
+                    ), get_changes("input/dir_replaced_with_link", joutput)
+                    assert any(
+                        chg["type"] == "mode" and chg["old_mode"].startswith("l")
+                        for chg in get_changes("input/link_replaced_by_file", joutput)
+                    )
```

Comment:
did you forget to add ", get_changes(...)" to the 2nd assert?

Reply:
It was a TODO to remove after the second commit. Will remove that.